base/setup/reactos/lang/bg-BG.rc

```diff
@@ -1,5 +1,7 @@
 LANGUAGE LANG_BULGARIAN, SUBLANG_DEFAULT
 
+/* Dialogs */
+
```

Comment:
For the record: I do think those are captain-obvious-comments of very questionable use, but I won't force you to run through all the languages to strip them again. Just mentioned for the next time you would consider adding them anywhere.